instrumentation/jdbc/library/build.gradle.kts

```diff
@@ -8,7 +8,7 @@ plugins {
 }
 
 dependencies {
-  implementation(project(":instrumentation-api"))
+  api(project(":instrumentation-api"))
```

Comment:
I think we add this with our plugin - can we just delete?

Reply:
Yeah, I think can delete it. Actually, I found another problem and thought this fix can fix it, but eventually found that problem is not here. Please check this issue: https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/3498

Reply:
You are right. Removed.